src/services/orders.ts

```diff
@@ -23,11 +26,13 @@ export const getAllOrdersAsUIOrders = async (baseToken: Token, quoteToken: Token
     const orders: SignedOrder[] = await getAllOrders(baseToken, quoteToken);
     try {
         const contractWrappers = await getContractWrappers();
-        const ordersAndTradersInfo = await contractWrappers.orderValidator.getOrdersAndTradersInfoAsync(
-            orders,
-            orders.map(o => o.makerAddress),
-        );
-        return ordersToUIOrders(orders, baseToken, ordersAndTradersInfo);
+        // HACK(dekz): Kovan DevUtils in contract-wrappers is currently incorrect
```

Comment:
🙈

Reply:
Is there some github / asana ticket we can point to where this will be fixed?

Reply:
This will soon be fixed with Amirs latest PR.